S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+    if(lang=='turkey'or lang=='az' or lang=='tr'):      #converts uppercase to lowercase
+        for letter in word:
+            if(letter=='i'or letter=="I"):
+                I=u"\u0131"
```

Comment:
what is u"\u0131" ? If a comment is added for explanation of, then it would be great.

Reply:
That certain piece of expression is the universal code or uni-code for the symbol 'Σ' but surely a comment would be great. Thanks!